Refit.Tests/RequestBuilder.cs

```diff
@@ -1058,7 +1062,7 @@ public void MethodsShouldBeCancellableDefault()
         {
             var fixture = new RequestBuilderImplementation<ICancellableMethods>();
             var factory = fixture.RunRequest("GetWithCancellation");
-            var output = factory(new object[0]);
+            var output = factory(Array.Empty<object>());
```

Comment:
Did not know this exists, but it looks like it's not even new. I assume it's cheaper than creating an empty array every time?

Reply:
Yup, removes allocations